bin/cqlsh.py

```
@@ -462,6 +469,14 @@ def __init__(self, hostname, port, color=False,
         self.tracing_enabled = tracing_enabled
         self.page_size = self.default_page_size
         self.expand_enabled = expand_enabled
+
+        if not consistency_level:
```

dse moved the code earlier in the init sequence, probably to fail faster